### PR TITLE
deployment/ccip/changeset: rm TestInitialDeploy

### PR DESCRIPTION
**Pull request by makramkd**
Ticket: https://smartcontract-it.atlassian.net/browse/CCIP-4516

This test is now a relic as we have more granular test cases in other test files.

### CODE REVIEW

**Review by mateusz-sekara — APPROVED** [review]

File: deployment/environment/devenv/README.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ Pre-requisites:
 
 #### Setting Up Testconfig with Simulated Private Ethereum Network
 
-To run tests (e.g., [ccip-test](../../integration-tests/smoke/ccip_test.go)), 
-you need to set up the testconfig following the [testconfig setup instructions](../../integration-tests/testconfig/README.md). 
+To run tests (e.g., [ccip-test](../../../integration-tests/ccip-tests/smoke/ccip_test.go)),
```
Comment (b-gopalswami) [inline deployment/environment/devenv/README.md:17]: The reference to ccip_test.go in this readme should be updated as you have removed it.

Comment (makramkd) [reply deployment/environment/devenv/README.md:17]: I believe this was referencing the <= v1.5 smoke test, not the v1.6 one

Comment (makramkd) [reply deployment/environment/devenv/README.md:17]: This reference was broken before, I just fixed it here

**Review by dimkouv — APPROVED** [review]

File: deployment/environment/devenv/README.md (hunk 1 above)
Comment (b-gopalswami) [reply deployment/environment/devenv/README.md:17]: I doubt that as <= v1.5 has its own readme. @AnieeG Tagging you as you have added this file initially.

**Review by connorwstein — APPROVED** [review]

**Pull request merged**